HT4_part1.py

```diff
@@ -0,0 +1,50 @@
+# import modules random and string
+import random
+import string
+
+def random_dicts():
+    j = 1
+    thisdict = {}
+    number_of_keys = random.randint(1, 28)
+    while j <= number_of_keys:
+        thisdict.update({random.choice(string.ascii_lowercase): random.randint(0, 100)})
+        j = j + 1
+    return(thisdict)
+
+def keys_list():
+    n = 0
+    list_of_keys = []                                                           # create empty list
+    while n < len(list_of_dicts):                                               # check if variable less than length (not including end, because already started from 0)
+        for k in list_of_dicts[n].keys():                                       # take every key one by one
+            if k not in list_of_keys:                                           # check if key is not exists in list of keys
+                list_of_keys.append(k)                                          # if not, add to the list
+        n = n + 1                                                               # increase variable to check next dict
+    return list_of_keys
+
+def dict_sum():
+    new_dict = {}                                                               # create new dict for result
+    for keys in keys_list():                                                    # checking all keys one by one

```

Comment:
maybe `for key in get_keys_list():`? To be more understandable for others who reads this code